content/blog/2021-06-10-on-ransomware-naming.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "On Ransomware Naming"
+created_at: 2021-06-10
+kind: article
+author_name: Richard "RichiH" Hartmann
+---
+
+As per Oscar Wilde, imitation is the sincerest form of flattery.
+
+The names "Prometheus" and "Thanos" have recently been taken up by a ransomware group. There's not much we can do about that except to inform you that this is happening. There's not much you can do either, except following the common supply chain & security practices. When deploying software, do it through one of those mechanisms:
```

Comment:
Should we link to one of the articles about it to give people more context?

Reply:
I am torn, but you are probably right